signale.js

```diff
@@ -90,6 +90,17 @@ class Signale {
   }
 
   _logger(type, ...messageObj) {
+    if (type === 'progress' && Object.prototype.hasOwnProperty.call(messageObj[0], 'label')) {
+      let base = '';
+      let i = 0;
+      for (; i < messageObj[0].current || 0; i++) {
```

Comment:
This could be too expensive when we call signale.progress({label:'Downloading', current: <CURRENT_VALUE>, total: <TOTAL_VALUE>}); inside any for loop or interval.

Reply:
I have to take a look at `node-progress` library first